sdc/tests/test_series.py

```diff
@@ -6013,35 +6013,40 @@ def test_series_islower_str(self):
     @skip_sdc_jit("Series.str.isalnum is not supported yet")
     def test_series_isalnum_str(self):
         cfunc = self.jit(isalnum_usecase)
-        for ser in test_global_stringmethods:
+        test_data = test_global_stringmethods + test_global_input_data_unicode_kind4
+        for ser in test_data:
```

Comment:
I vote for renaming the variable `ser` (e.g. to `data`), because `test_data` doesn't contain `Series`.